### DISCUSSION TITLE
Part guidelines/what and what not/coloring suggestion

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#7] [Collaborator Disscusion]
VERY_COOL_PLACEHOLDER

**Comment by likeproblem** [comment]
Just in case I have special folder "Common" for repeated different elements, such as hatches

**Comment by likeproblem** [comment]
@UncleMateo the batteries seem little off... sorry if it sounds like complaining, but it is little bit off, I will try to fix them by myself
![screenshot51](https://github.com/user-attachments/assets/0154d18d-b170-4bd7-963a-f771e7c7e221)
The battery clippers on small battery *should* be colored like his big brother, and some parts such as this stack battery has tiny green lines, maybe you went by accident over one pixel?

And stack smallest gyroscope for some reason doesn't have colored stripes

(again sorry for "complaining", just feels off)

**Reply by UncleMateo** [reply]
No that's totally cool, I take no offense.  Sometimes a second pair of eyes catches the things the first pair unintentionally learned to ignore.

**Comment by Spartwo** [comment]
Red is the primary colour

**Comment by likeproblem** [comment]
@EmberSkyMedia ladderz, so the issue I found bunch of meshes go outside here and there, AA in some places(it is still noticeble in some places, which is not that good, but that spot is very heavy).

Also please color the yellow ladder, thanks in advance :)
